setup.sh

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
+exec 2>/var/log/VMInit.log 2>&1
```

Comment:
This should be put under the root user check. ```/var/log``` isn't writable by non root users (as much as I know)